main/inc/ajax/model.ajax.php

```diff
@@ -425,7 +425,7 @@ function getWhereClause($col, $oper, $val)
     $obj->delete($_REQUEST['id']);
 }
 
-$is_allowedToEdit = api_is_allowed_to_edit(null, true) || api_is_allowed_to_edit(true) || api_is_drh();
+$is_allowedToEdit = api_is_allowed_to_edit(null, true) || api_is_allowed_to_edit(true) || api_is_drh() || api_is_student_boss();
```

Comment:
I modified this to avoid giving permissions to student boss in too many (ajax) cases